src/main/java/pl/futurecollars/invoicing/db/Database.java

```diff
@@ -30,5 +30,9 @@ default BigDecimal visit(
         .map(invoiceEntryToValue)
         .reduce(BigDecimal.ZERO, BigDecimal::add);
   }
+
+  default void reset() {
```

Comment:
There could be just delete all method, but for now this can stay, it should do the job, but won't be efficient with more data.